xbmc/cores/DllLoader/Win32DllLoader.cpp

```diff
@@ -324,6 +324,9 @@ void Win32DllLoader::OverrideImports(const std::string &dll)
 
 bool Win32DllLoader::NeedsHooking(const char *dllName)
 {
+  if (!StringUtils::EndsWithNoCase(dllName, "libdvdread.dll"))
```

Comment:
libdvdread was my guess from memory about the libdvd project layout but dvdread maybe is statically linked into the other two?

would it make sense to add some logging into our emulation layer into nightlies and ask people to upload logs so we get a better picture of what might be using it that we may be missing? /note, not suggesting automatic uploads as that's some work and heavily disliked